### ISSUE TITLE
Don't assume which traits are in scope

### ISSUE THREAD

**Issue by RReverser** [panicbit/quickcheck_derive#7]
Currently getting

```
[rustc]
no method named `gen_range` found for type `&mut G` in the current scope

help: items from traits can only be used if the trait is in scope
note: the following trait is implemented but not in scope, perhaps add a `use` for it:
      `use rand::Rng;`
```

on any `#[cfg_attr(test, derive(Arbitrary))]` because generated code seems to assume that traits are already in scope with `use ...` and calls `g.gen_range()` directly. Instead, it should specify absolute path to each used trait.

**Comment by RReverser** [comment]
~~Judging by the code, same fixes should be done for `impl Arbitrary`, `g: Gen` etc.~~

**Comment by panicbit** [comment]
Thanks! I'll take care of it :smiley:

**Comment by RReverser** [comment]
Actually, the latter is probably not necessary, I see `use` statements are already generated. Weird that they don't help with inherited `RngCore` methods.

**Comment by RReverser** [comment]
Ohh... it just uses outdated quickcheck where it wasn't necessary.

**Comment by RReverser** [comment]
This is all weird... So, apparently, one has to have `rand` crate now in addition to quickcheck to get access to these methods like `gen_range` which derive uses.

**Comment by panicbit** [comment]
It looks like quickcheck 0.7 changed the bound on `Gen` to be `RngCore` instead of `Rng`,
meaning that the `gen_range` method won't be available anymore without explicitly adding that bound.
Seems like there's some updating to do.

**Comment by RReverser** [comment]
Yeah. I made a PR to fix this (see above).

**Comment by panicbit** [comment]
I've released 0.2.0 on crates.io with the update.

**Comment by RReverser** [comment]
Thanks!
